spring-boot-actuator/pom.xml

```diff
@@ -138,6 +138,11 @@
 			<optional>true</optional>
 		</dependency>
 		<dependency>
+			<groupId>org.hibernate</groupId>
+			<artifactId>hibernate-entitymanager</artifactId>
```

Comment:
@rajadileepkolli that PR hasn't been reviewed in the target branch yet. We'll take care of that at that time.